clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -1413,6 +1414,24 @@ void EmitAssemblyHelper::RunOptimizationPipeline(
             FPM.addPass(BoundsCheckingPass());
           });
 
+    if (LangOpts.Sanitize.has(SanitizerKind::LKMMDepChecker)) {
+      if (CodeGenOpts.OptimizationLevel == 0) {
+        PB.registerOptimizerLastEPCallback(
+            [](ModulePassManager &MPM, OptimizationLevel Level) {
+              MPM.addPass(AnnotateLKMMDeps());
```

Comment:
For debugging?

Reply:
Yup. But I just saw there's another workaround: `clang -O1 -mllvm -disable-llvm-optzns -disable-llvm-passes`. If I use that instead, I can get rid of the above.